### ISSUE TITLE
Changing default terminal size at build-time

### ISSUE THREAD

**Issue by outpaddling** [lxqt/qterminal#291]
I'm maintaining a qterminal package with a couple of enhancements that set the default font and terminal size.

In 0.6.0, I used the following patch to qterminal.ui:

```
--- src/forms/qterminal.ui.orig 2014-10-22 08:13:14 UTC
+++ src/forms/qterminal.ui
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>600</width>
-    <height>420</height>
+    <width>820</width>
+    <height>597</height>
    </rect>
   </property>
   <property name="windowTitle">
```

This no longer works in 0.7.0.  Looking at qterminal.ini, I think it's because it no longer uses the width and height properties and now uses MainWindow/size.

Is there a similar patch I could use to achieve the same effect in 0.7.0?

Is qterminal.ui out of date?

Thanks,

    Jason

**Comment by outpaddling** [comment]
Thanks, the reduction in dependencies is good news and might render my concern moot.

Also, I created an ticket regarding default terminal size:

https://github.com/lxde/qterminal/issues/291

I've tried solving this myself by playing with changes to the ui file and with QT 5 Designer, but I'm green on QT development and have not met success yet.  If I don't figure this out soon, we can commit the port as-is for now.  I find the problem very bothersome, but it's ultimately only cosmetic.
